<?xml version="1.0"?>
<launch>

  <arg name="num_samples" default="100000" />
  <arg name="chain_start" default="base" />
  <arg name="chain_end" default="ee_link" />
  <arg name="timeout" default="0.05" />
  <!-- $(find xacro)/xacro - -inorder '$(find ur_e_description)/urdf/ur5e_robot.urdf.xacro'-->
  <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find trac_ik_examples)/urdf/ur5e_with_hand_fused.urdf.xacro'" />

  <node name="trac_ik_tests" pkg="trac_ik_examples" type="ik_tests" output="screen">

    <param name="num_samples" value="$(arg num_samples)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="/robot_description"/>

  </node>

</launch>
